client/js/views/Home.vue

```diff
@@ -109,9 +164,11 @@ export default {
   data() {
     return {
       newGameName: '',
+      newGameIsRanked: false,
```

Comment:
What about making newGame an object with two keys, `name` and `ranked`?

Reply:
Why? Same issue could apply in the new `CreateGameDialog` -- not clear to me what the pros and cons are of one object vs separate params.